google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -124,19 +121,19 @@ def accept(self, visitor, stack=None):
             try:
                 for resource in yielder.iter():
                     res = resource
+                    new_stack = stack + [self]
 
-                    def call_accept():
-                        res.accept(visitor, stack + [self])
-
-                    if res.is_leaf():
-                        call_accept()
+                    if res.type() != 'project':
```

Comment:
Nit: If we could make this a res.should_dispatch() with a base implementation of "return False" and a Project class implementation of "return True", that would be better documentation of what's going on.

Reply:
Fixed.